README.md

```diff
@@ -2,7 +2,7 @@
 
 # [OnchainKit](https://github.com/coinbase/onchainkit/)
 
-> OnchainKit is a collection of tools to build world-class onchain apps with CSS, React, and Typescript.
+> OnchainKit is a collection of tools to build world-class onchain apps with NextJS.
```

Comment:
Right now we have a hard dependency on NextJS, so it makes sense to mention that up front

Reply:
I don't think NextJS should be a hard dependency. I do believe we should actually change our code to make it easier to use in any Node / TS enviroment.

Reply:
Agreed that the goal should be a universal library.